bundles/org.openhab.binding.benqprojector/README.md

```diff
@@ -55,16 +55,16 @@ Some notes:
 
 ## Channels
 
-| Channel            | Item Type | Purpose                                             | Values    |
-| ------------------ | --------- | --------------------------------------------------- | --------- |
-| power              | Switch    | Powers the projector on or off.                     |           |
-| source             | String    | Retrieve or set the input source.                   | See above |
-| picturemode        | String    | Retrieve or set the picture mode.                   | See above |
-| aspectratio        | String    | Retrieve or set the aspect ratio.                   | See above |
-| freeze             | Switch    | Turn the freeze image mode on or off.               |           |
-| blank              | Switch    | Turn the screen blank mode on or off.               |           |
-| directcmd          | String    | Send a command directly to the projector.           | Send only |
-| lamptime           | Number    | Retrieves the lamp hours.                           | Read only |
+| Channel            | Item Type | Purpose                                             | Values     |
+| ------------------ | --------- | --------------------------------------------------- | ---------- |
+| power              | Switch    | Powers the projector on or off.                     |            |
+| source             | String    | Retrieve or set the input source.                   | See above  |
+| picturemode        | String    | Retrieve or set the picture mode.                   | See above  |
+| aspectratio        | String    | Retrieve or set the aspect ratio.                   | See above  |
+| freeze             | Switch    | Turn the freeze image mode on or off.               |            |
+| blank              | Switch    | Turn the screen blank mode on or off.               |            |
+| directcmd          | String    | Send a command directly to the projector.           | Write only |
+| lamptime           | Number    | Retrieves the number of lamp hours used.            | Read only  |
```

Comment:
I am definitively not fluent in English but I have a doubt if this is good English.
In another PR, you used "Retrieves the number of hours the lamp has been used".
Just ignore this remark if this is good English.

Maybe you should also adjust the channel description.

Reply:
I updated the wording and applied it to the channel description also. Thanks!